packages/alert-dialog/index.d.ts

```diff
@@ -0,0 +1,20 @@
+import { DialogProps, DialogContentProps } from "@reach/dialog";
+
+declare module "@reach/alert-dialog" {
+  type AlertDialogProps = {
+    isOpen: boolean;
```

Comment:
According to [documentation](https://ui.reach.tech/alert-dialog#alertdialog-isopen) `isOpen` is by default `true` . Shouldn't we make it optional?

Reply:
ah yeah.  ill go through and mark all the optional ones.